go.mod

```diff
@@ -19,7 +19,7 @@ require (
 	github.com/benbjohnson/clock v1.3.0
 	github.com/bradleyjkemp/cupaloy v2.3.0+incompatible
 	github.com/elastic/go-elasticsearch/v7 v7.17.1
-	github.com/estuary/flow v0.1.9-0.20230405122715-43db16b1b0cb
+	github.com/estuary/flow v0.2.4-0.20230622152821-5fad88ecf0df
```

Comment:
TODO: Update this if needed after https://github.com/estuary/flow/pull/1091 merges, prior to merging this.